spec/unit/ssl/certificate_request_spec.rb

```diff
@@ -177,6 +177,17 @@
       end
     end
 
+    context "with DNS and IP SAN specified" do
+      before :each do
+        Puppet[:dns_alt_names] = ""
+      end
+
+      it "should return the subjectAltName values" do
+        request.generate(key, :dns_alt_names => 'DNS:foo, bar, IP:172.16.254.1')
```

Comment:
Similar question here, should we rename this option to `:subject_alt_names` as well? (this one is only internal, not user-facing)